### PR TITLE
Fandom master

### PR DESCRIPTION
**Pull request by mmeller-wikia**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

**Review by mmeller-wikia — COMMENTED** [review]
Note: My assumption is that `internal/httpclient` is automatically generated -> I am skipping these files

File: identity/handler.go [hunk 1]
```diff
@@ -524,3 +590,70 @@ func (h *Handler) delete(w http.ResponseWriter, r *http.Request, ps httprouter.P
...
+//	Schemes: http, https
+//
+//	Security:
+//	  oryAccessToken:
+//
+//	Responses:
+//	  204: emptyResponse
+//	  404: jsonError
+//	  500: jsonError
+func (h *Handler) deleteMultifactorCredential(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```
Comment (mmeller-wikia) [inline identity/handler.go:632]: Issue: This functionality is not migrated

File: driver/registry_default_settings.go
Comment (mmeller-wikia) [inline driver/registry_default_settings.go:1]: Note: These changes are not migrated

File: identity/handler.go (hunk 1 above)
Comment (mmeller-wikia) [reply identity/handler.go:632]: See https://github.com/Wikia/kratos/pull/97/commits/909636d21142ee4a4dfeb2f5c92dfb1cea2611ed

File: driver/registry_default.go [hunk 2]
```diff
@@ -199,7 +205,9 @@ func (m *RegistryDefault) RegisterRoutes(ctx context.Context, public *x.RouterPu
 }
 
 func NewRegistryDefault() *RegistryDefault {
-	return &RegistryDefault{}
+	return &RegistryDefault{
+		rc: map[string]*retryablehttp.Client{},
+	}
 }
```
Comment (mmeller-wikia) [inline driver/registry_default.go:211]: Note: Not migrated

File: selfservice/hook/web_hook.go
Comment (mmeller-wikia) [inline selfservice/hook/web_hook.go:1]: Note: migrated in https://github.com/Wikia/kratos/pull/97

File: identity/handler.go (hunk 1 above)
Comment (adpaste) [reply identity/handler.go:632]: It was replaced by the upstream implementation.
https://fandom.atlassian.net/browse/PLATFORM-9038

File: driver/registry_default_settings.go
Comment (adpaste) [reply driver/registry_default_settings.go:1]: Fixed that.

**Pull request closed without merging**